transport/src/main/java/io/netty/channel/socket/DefaultDatagramChannelConfig.java

```diff
@@ -44,7 +44,7 @@
 
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(DefaultDatagramChannelConfig.class);
 
-    private final DatagramSocket javaSocket;
+    protected final DatagramSocket javaSocket;
```

Comment:
can we add a final accessor instead of exposing the member variable for public classes ... allows us more flexibility later and the JIT should inline it anyways.